src/main/java/dev/latvian/mods/kubejs/plugin/builtin/wrapper/TextWrapper.java

```diff
@@ -239,6 +239,16 @@ static MutableComponent translate(String key, Object... objects) {
 		return Component.translatable(key, objects);
 	}
 
+	@Info("Returns a translatable component of the input key, with args of the objects and a fallback")
```

Comment:
It might be worth saying 'and a fallback translation in case the client does not have one', although that is very lengthy 😕